liota/entities/devices/device.py

```diff
@@ -59,11 +59,11 @@ def __init__(self, name, parent, entity_id, entity_type="Device"):
             entity_type=entity_type
         )
 
-    def register(self, entity_obj, dcc_obj, reg_entity_id):
+    def register(self, dcc_obj, reg_entity_id):
```

Comment:
Ideally I should have commented this in Winnie's review. Somehow, the register() call in the base class like device seems to be misplaced. We separated out the entities and registered_entities. Now we are having the register() call in entity?